client/my-sites/store/style.scss

```diff
@@ -63,6 +63,74 @@
 			margin: 0;
 		}
 	}
+
+	// Overwrite Tooltip styles
```

Comment:
Would it be possible to modularize these styles into a separate SCSS file within `client/my-sites/stats`? If so, we'd be able to include them in this file with an `@import` invocation.

Reply:
Yes! I tried it for a while and decided to place it on the `root of Stats` rather than the component `Chart`. However, since the `Chart` has been used in `Stats only`, we could move these styles back to `Chart` when all usages are refactored.

Reply:
Resolved in [b9bf4d1](https://github.com/Automattic/wp-calypso/pull/69661/commits/b9bf4d119e8fe789b75008a039c49496d763c02f).